docs/up-ga/downloads/v1.md

```diff
@@ -11,13 +11,68 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-    No data available for the specified measure.
+  Total downloads of the landing page of a work, as determined by Google Analytics of the website. To count multiple downloads of the same landing page by the same user, there must be a gap of at least 30 minutes from the last download of that landing page, even if that previous download was not counted.
```

Comment:
remove `of the website`.

A landing page is not downloaded, a file is; change to:
`To count multiple downloads of the same file by the same user`